ECommerceAPI/Services/EmailServices/Implementations/EmailService.cs

```diff
@@ -31,7 +27,12 @@ public async Task SendConfirmationEmailAsync(string to,string confirmationLink)
         var body = EmailMessageGenerator.GenerateConfirmEmailMessageBody(confirmationLink);
         await SendEmailAsync(to, subject, body);
     }
-    
+    public async Task SendResetPasswordEmailAsync(string to, string resetLink)
+    {
+        var subject = "Reset Password";
+        var body = EmailMessageGenerator.GenerateForgotPasswordMessageBody(resetLink);
```

Comment:
since it helper, lets say okay to use static but in real life the static methods not very good since it need more testing effort in unit test